tree/treeplayer/inc/TTreeFormula.h

```diff
@@ -177,6 +177,11 @@ friend class TTreeFormulaManager;
    Int_t       DefinedVariable(TString &variable, Int_t &action) override;
    virtual TClass*     EvalClass() const;
 
+   ////////////////////////////////////////////////////////////////////////////
+   /// \brief Evaluate this treeformula
```

Comment:
Thanks! Please move this Doxygen block instead to TTreeFormula.cxx line 3927 to prevent duplicities.